R/project-tool.R

```diff
@@ -85,6 +85,12 @@ getScriptCode <- function(input, analyses)
         {
             addComment("Load analysis table", condition = comment)
             addCall(anaInfoVarName, "read.csv", list(value = anaTableFile, quote = TRUE))
+            if (any(anaTable$exclude))
```

Comment:
I think it would be nicer if the actual csv doesn't contain the ignored files, i.e. by excluding it when calling `write.csv`.